### PR TITLE
docs: added a new built-in function `getenv`

### PR DESCRIPTION
**Pull request by lafirest**
https://github.com/emqx/emqx/pull/13507

### CODE REVIEW

**Comment by zmstone** [comment]
added some limitations here: https://github.com/emqx/emqx/pull/13540
please update doc accordingly .

File: en_US/configuration/configuration.md [hunk 1]
```diff
@@ -524,6 +524,10 @@ Below are the functions that can be used in the expressions:
   - `str_gte(A, B)`: Return 'true' if A is not before B in lexicographic order, otherwise 'false'.
   - `str_lt(A, B)`: Return 'true' if A is before B in lexicographic order, otherwise 'false'.
   - `str_lte(A, B)`: Return 'true' if A is not after B in lexicographic order, otherwise 'false'.
+- **System functions**:
+  - `getenv(Name)`: Return the value of the environment variable `Name` with the following limitations:
```
Comment (Meggielqk) [inline en_US/configuration/configuration.md]:
```suggestion
  - `getenv(Name)`: Return the value of the environment variable `Name` with the following constraints:
```

File: en_US/data-integration/rule-sql-builtin-functions.md [hunk 2]
```diff
@@ -1486,6 +1484,11 @@ Generates a version 4 UUID without hyphens. Example:
 uuid_v4_no_hyphen() = 'd7a39aa4195a42068b962eb9a665503e'
 ```
 
-## <!--Schema Registry and Sparkplug B Functions-->
+## System Function
+
+### getenv(Name)
+
+Return the value of the environment variable `Name` with the following limitations:
```
Comment (Meggielqk) [inline en_US/data-integration/rule-sql-builtin-functions.md]:
```suggestion
Return the value of the environment variable `Name` with the following constraints:
```

File: zh_CN/configuration/configuration.md [hunk 3]
```diff
@@ -525,6 +525,10 @@ EMQX 包含一系列丰富的字符串、数组、随机和散列函数，类似
   - `str_gte(A, B)`：如果 A 在字典顺序上不位于 B 之前，则返回 'true'，否则返回 'false'。
   - `str_lt(A, B)`：如果 A 在字典顺序上位于 B 之前，则返回 'true'，否则返回 'false'。
   - `str_lte(A, B)`：如果 A 在字典顺序上不位于 B 之后，则返回 'true'，否则返回 'false'。
+- **系统函数**:
+  - `getenv(Name)`：返回环境变量 `Name` 的值，注意以下限制：
```
Comment (Meggielqk) [inline zh_CN/configuration/configuration.md]:
```suggestion
  - `getenv(Name)`：返回环境变量 `Name` 的值，并遵循以下限制：
```

File: zh_CN/data-integration/rule-sql-builtin-functions.md [hunk 4]
```diff
@@ -1479,3 +1479,11 @@ uuid_v4() = 'f5bb7bea-a371-4df7-aa30-479add04632b'
 uuid_v4_no_hyphen() = 'd7a39aa4195a42068b962eb9a665503e'
 ```
 
+## 系统函数
+
+### getenv(Name)
+
+返回环境变量 `Name` 的值，注意以下限制：
```
Comment (Meggielqk) [inline zh_CN/data-integration/rule-sql-builtin-functions.md]:
```suggestion
返回环境变量 `Name` 的值，并遵循以下限制：
```

**Review by zmstone — APPROVED** [review]

**Pull request merged**